src/components/App/App.component.jsx

```diff
@@ -7,12 +7,14 @@ import LoginPage from '../../pages/Login';
 import NotFound from '../../pages/NotFound';
 import SecretPage from '../../pages/Secret';
 import Private from '../Private';
-import Fortune from '../Fortune';
+// import Fortune from '../Fortune';
```

Comment:
It is not a good practice to leave commented code, specially if you won't use it in the future. If you would need it, it would be great to add a comment for others to have a little of context, if not, just delete it.